the-shore/patterns/home.php

```diff
@@ -0,0 +1,205 @@
+<!-- /wp:image --></div>
+<!-- /wp:group --></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns -->
+
+<!-- wp:spacer {"height":"50px"} -->
+<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer --></div>
+<!-- /wp:group -->
+
+<!-- wp:cover {"url":"<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/hotel_image_6.jpg","dimRatio":20,"overlayColor":"black","isUserOverlayColor":true,"minHeight":600,"minHeightUnit":"px","style":{"spacing":{"padding":{"right":"6vw","left":"6vw"}}},"layout":{"type":"default"}} -->
+<div class="wp-block-cover" style="padding-right:6vw;padding-left:6vw;min-height:600px"><span aria-hidden="true" class="wp-block-cover__background has-black-background-color has-background-dim-20 has-background-dim"></span><img class="wp-block-cover__image-background" alt="<?php echo __('', 'the-shore');?>" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/hotel_image_6.jpg" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"style":{"typography":{"textTransform":"uppercase","letterSpacing":"0.1em"}},"fontSize":"x-small","fontFamily":"commissioner"} -->
+<p class="has-commissioner-font-family has-x-small-font-size" style="letter-spacing:0.1em;text-transform:uppercase"><?php echo __('Bars &amp; Restaurant', 'the-shore');?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"textAlign":"left","style":{"typography":{"fontStyle":"normal","fontWeight":"300"}},"fontSize":"large"} -->
+<h2 class="wp-block-heading has-text-align-left has-large-font-size" style="font-style:normal;font-weight:300"><?php echo __('Enjoy a cocktail at our stylish bar, or savor a fine dining experience with a backdrop of stunning ocean views.', 'the-shore');?></h2>
+<!-- /wp:heading -->
+
+<!-- wp:buttons -->
+<div class="wp-block-buttons"><!-- wp:button {"textColor":"base","style":{"elements":{"link":{"color":{"text":"var:preset|color|base"}}}},"className":"is-style-outline"} -->
+<div class="wp-block-button is-style-outline"><a class="wp-block-button__link has-base-color has-text-color has-link-color wp-element-button"><?php echo __('Discover Bars &amp; Restaurant', 'the-shore');?></a></div>
+<!-- /wp:button --></div>
+<!-- /wp:buttons --></div></div>
+<!-- /wp:cover -->
+
+<!-- wp:group {"metadata":{"name":"Banner with gallery"},"style":{"spacing":{"blockGap":"2.8rem"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:spacer {"height":"50px"} -->
+<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:image {"width":"380px","sizeSlug":"full","linkDestination":"none","align":"center"} -->
+<figure class="wp-block-image aligncenter size-full is-resized"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/line-1.png" alt="<?php echo __('', 'the-shore');?>" style="width:380px"/></figure>
```

Comment:
This needs alt text.

Reply:
This is a separator, I think it's fine without any alt text.